pkg/auth/model/model.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/auth/model";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.auth.model;
+
+message UserData {
+    int64 ID = 1;
+    google.protobuf.Timestamp CreatedAt = 2;
+    string Username = 3;
+    // FriendlyName, if set, is a shorter name for the user than
+    // Username.  Unlike Username it does not identify the user (it
+    // might not be unique); use it in the user's GUI rather than in
+    // backend code.
+    string FriendlyName = 4;
+    string Email = 5;
+    bytes EncryptedPassword = 6;
```

Comment:
What about source? Do we need to save this information?

Reply:
done, thanks for the catch